lib/ssl/src/dtls_gen_connection.erl

```diff
@@ -545,10 +545,9 @@ handle_info({CloseTag, Socket}, StateName,
     %% with widespread implementation practice.
```

Comment:
https://github.com/erlang/backlog/issues/130
but this seems to be a private repo

Reply:
got it, so the branch name should be changed then.

Reply:
Well yes, to avoid any confusion.  I think you could change the name of your branch and later (when next step feels ready) make a new PR replacing this one.